jans-auth-server/client/src/test/java/io/jans/as/client/ciba/BackchannelAuthenticationPingMode.java

```diff
@@ -528,11 +513,7 @@ public void backchannelTokenDeliveryModePingIdTokenHintRS256(
         RegisterResponse registerResponse = registerClient.exec();
 
         showClient(registerClient);
-        assertEquals(registerResponse.getStatus(), 201, "Unexpected response code: " + registerResponse.getEntity());
-        assertNotNull(registerResponse.getClientId());
-        assertNotNull(registerResponse.getClientSecret());
-        assertNotNull(registerResponse.getRegistrationAccessToken());
-        assertNotNull(registerResponse.getClientSecretExpiresAt());
+Asserter.assertOk(registerResponse);
```

Comment:
Not formatted (as well as in many other places).
Lets use static import to have just `assertOk(registerResponse);`.
Otherwise nice changes. It makes our tests much shorter.

Reply:
OK, it was fixed